tools/system_libs.py

```diff
@@ -264,6 +249,41 @@ def create_wasm_compiler_rt(libname):
     run_commands([[shared.LLVM_AR, 'cr', '-format=gnu', lib] + o_s])
     return lib
 
+  def files_in_path(path_components=None, filenames=None):
+    assert path_components is not None and filenames is not None
```

Comment:
Why have default arguments at all if you're just going to assert that neither of them are used?